openmdao/core/tests/test_group.py

```diff
@@ -226,6 +318,42 @@ def test_group_rename_connect(self):
         self.assertEqual(p['comp1.b'], 6.0)
         self.assertEqual(p['comp2.b'], 9.0)
 
+    def test_promotes_any(self):
+        import openmdao.api as om
+
+        class SimpleGroup(om.Group):
+
+            def setup(self):
+
+                self.add_subsystem('comp1', om.IndepVarComp('x', 5.0))
+                self.add_subsystem('comp2', om.ExecComp('b=2*a'))
+
+            def configure(self):
+                self.promotes('comp1', any=['*'])
+
+        top = om.Problem(model=SimpleGroup())
+        top.setup()
+
+        self.assertEqual(top['x'], 5)
+
+    def test_promotes_inputs_and_outputs(self):
```

Comment:
Tests that are used in the feature docs need to include all imports.  If you forget to do this, it should fail when the docs build (tests are run independently in that process), so if you built these docs, something isn't working right there.

Reply:
Yeah I forgot. This was added in the latest commit